CRM/Core/BAO/MessageTemplate.php

```diff
@@ -369,16 +370,17 @@ public static function sendTemplate($params) {
     $params['html'] = $mailContent['html'];
 
     if ($params['toEmail']) {
-      $contactParams = [['email', 'LIKE', $params['toEmail'], 0, 1]];
-      [$contact] = CRM_Contact_BAO_Query::apiQuery($contactParams);
-
-      $prefs = array_pop($contact);
-
-      if (isset($prefs['preferred_mail_format']) and $prefs['preferred_mail_format'] === 'HTML') {
+      // @todo - consider whether we really should be loading
+      // this based on 'the first email in the db that matches'.
+      // when we likely have the contact id. OTOH people probably barely
+      // use preferredMailFormat these days - the good fight against html
+      // emails was lost a decade ago...
```

Comment:
I would fully support ripping out 'preferred_mail_format'.

Reply:
@jmcclelland given you just put up a PR about importing this field I'm thinking you might still see value in it? I am a bit confused why we wouldn't always send 'BOTH' and let the mail client decide

Reply:
I don't think sending in non-html ever took off. I'd be in favour of obsoleting then later removing this.